src/components/routers/Webhook.tsx

```diff
@@ -175,8 +179,9 @@ export class WebhookForm extends NodeRouterForm<WebhookProps, WebhookState> {
             <div>
                 <h4 className={styles.headers_title}>Headers</h4>
                 <p>Add any additional headers belows that you would like to send along with your request.</p>
-                <FlipMove enterAnimation="fade" leaveAnimation="fade" className={styles.actions} duration={300} easing="ease-out">
+                <FlipMove easing="ease-out" enterAnimation="accordionVertical" leaveAnimation="accordionVertical" className={styles.headers} duration={300}>
                     {headerElements}
+                    <div style={{ height: "25px" }} />
```

Comment:
On exit animation, the container instantly shrinks in height causes the post form below to collide. One option is making the postForm the last element which results in nice enters / exits but causes the animation to trigger when switching between GET and POST on the main pane, so just adding some spacing for now.